packages/@glimmer/opcode-compiler/lib/syntax/expressions.ts

```diff
@@ -57,23 +56,7 @@ EXPRESSIONS.add(SexpOpcodes.GetStrictFree, (op, [, sym, _path]) => {
   });
 });
 
-EXPRESSIONS.add(SexpOpcodes.GetFreeAsFallback, (op, [, freeVar, path]) => {
-  op(HighLevelResolutionOpcode.ResolveLocal, freeVar, (name: string, moduleName: string) => {
-    if (DEBUG) {
-      let propertyPath = path ? [name, ...path].join('.') : name;
-
-      deprecate(
-        `The \`${propertyPath}\` property path was used in the \`${moduleName}\` template without using \`this\`. This fallback behavior has been deprecated, all properties must be looked up on \`this\` when used in the template: {{this.${propertyPath}}}`,
-        false,
-        {
-          id: 'this-property-fallback',
-        }
-      );
-    }
-
-    op(Op.GetVariable, 0);
-    op(Op.GetProperty, name);
-  });
+EXPRESSIONS.add(SexpOpcodes.GetFreeAsFallback, (op, [, , path]) => {
   withPath(op, path);
 });
```

Comment:
Should this whole thing be ✂️?

Reply:
I can't tell if `GetFreeAsFallback` also handles the case of an in scope "loose/free" variable.  @rwjblue

Reply:
@snewcomer - Let's land this as is now, and maybe you can send a follow up PR that removes the `GetFreeAsFallback` op code and see if all tests pass...